cfme/infrastructure/config_management.py

```diff
@@ -36,97 +37,26 @@
-            {
-                'infrastructure_config_manager_new':
-                lambda _: cfg_btn('Add a new Provider'),
-                'infrastructure_config_manager':
-                [
-                    lambda ctx: sel.check(
-                        Quadicon(
-                            '{} Configuration Manager'
-                            .format(ctx['manager'].name), None).checkbox()),
-                    {
-                        'infrastructure_config_manager_edit':
-                        lambda _: cfg_btn('Edit Selected item'),
-                        'infrastructure_config_manager_refresh':
-                        lambda _: cfg_btn('Refresh Relationships and Power states',
-                                  invokes_alert=True),
-                        'infrastructure_config_manager_remove':
-                        lambda _: cfg_btn('Remove selected items from the VMDB', invokes_alert=True)
-                    }
-                ],
-                'infrastructure_config_manager_detail':
-                [
-                    lambda ctx: sel.click(
-                        Quadicon('{} Configuration Manager'.format(ctx['manager'].name), None)),
-                    {
-                        'infrastructure_config_manager_edit_detail':
-                        lambda _: cfg_btn('Edit this Provider'),
-                        'infrastructure_config_manager_refresh_detail':
-                        lambda _: cfg_btn('Refresh Relationships and Power states',
-                                  invokes_alert=True),
-                        'infrastructure_config_manager_remove_detail':
-                        lambda _: cfg_btn('Remove this Provider from the VMDB', invokes_alert=True),
-                        'infrastructure_config_manager_config_profile':
-                        lambda ctx: (tb.select('List View'),
-                               page.list_table_config_profiles.click_cell(
-                            'Description', ctx['profile'].name))
-                    }
-                ]
-            }
-        ],
-        'infrastructure_config_systems':
-        [
-            lambda _: accordion.tree('Configured Systems',
-                version.pick({version.LOWEST: 'All Red Hat Satellite Configured Systems',
-                              '5.6': 'All Configured Systems'})),
-            {
-                'infrastructure_config_system':
-                [
-                    lambda ctx: (tb.select('Grid View'),
-                    sel.click(Quadicon(ctx['system'].name, None))),
-                    {
-                        'infrastructure_config_system_provision':
-                        lambda _: cfg_btn('Provision Configured System'),
-                        'infrastructure_config_system_edit_tags':
-                        lambda _: cfg_btn('Edit Tags')
-                    }
-                ]
-            }
-        ]
-    }
-)
 
+def page_match_controller(title):
```

Comment:
maybe a bit more generic of a name since its matching more than the controller. page_match_elements()?